src/app/components/Sword.js

```diff
@@ -0,0 +1,34 @@
+export const Sword = () => {
```

Comment:
Brak testow?

Reply:
Reszta spoko chyba

Reply:
@DomiZet no testy by się przydały... raczej nie zajmą dużo czasu, bo chyba tylko do sprawdzenia jest, czy to się renderuje i faktycznie jest.... Zobacz sobie na testy Tomka z jego PR'a,, bo tam takie rzeczy są dokładnie sprawdzane :-)